tests/drivers/rtc/rtc_api/testcase.yaml

```diff
@@ -10,4 +11,9 @@ tests:
     filter: dt_alias_exists("rtc")
     depends_on: rtc
     platform_exclude:
+      - b_u585i_iot02a
+      - nucleo_f767zi
+      - nucleo_h563zi
+      - nucleo_h723zg
+      - nucleo_u575zi_q
```

Comment:
Excluding the added boards is not required. They can not be run, so they will only be built, which is a good test in itself :) The reason `qemu_x86_64` is excluded is because it can be run, but runs at real-time, timing out the test suite :)

Reply:
Of crs, sorry, I just didn't pay attention it's exclude & not include.